### ISSUE TITLE
Bigger LLaMA models

### ISSUE THREAD

**Issue by alexl83** [tatsu-lab/stanford_alpaca#38]
Dear Stanford Researchers, Professors, Students (all geniuses) thank you for your amazing job!
Would the tuning code you released in this repo (and the dataset) be fit for finetuning larger LLaMA models like 13b/30b/65b?

How would the computational effort scale with such models?

**Comment by rtaori** [comment]
We don't have official support for larger training, but long as you can add more GPUs, should just be a matter of switching out the model path and tuning batch size & learning rate.
